src/main/drivers/timer_def_stm32h7xx.h

```diff
@@ -64,7 +64,11 @@
 #define DEF_TIM_DMA__BTCH_TIM4_CH1    DEF_TIM_DMA_FULL(DMA_REQUEST_TIM4_CH1)
 #define DEF_TIM_DMA__BTCH_TIM4_CH2    DEF_TIM_DMA_FULL(DMA_REQUEST_TIM4_CH2)
 #define DEF_TIM_DMA__BTCH_TIM4_CH3    DEF_TIM_DMA_FULL(DMA_REQUEST_TIM4_CH3)
+#ifdef USE_DSHOT_DMAR
+#define DEF_TIM_DMA__BTCH_TIM4_CH4    DEF_TIM_DMA_FULL(DMA_REQUEST_TIM4_UP)
```

Comment:
i am still not in love with this, but as correctly noted as long as we use USE_DSHOT_DMAR it does not matter that the request does not match the channel as long as there is a valid dmaTag available.

I suggest we add a comment here, noting that this is technically not the correct thing to do.